lib/dts-lib.robot

```diff
@@ -0,0 +1,130 @@
+    [Documentation]    Keyword allows to run EC Transition procedure in the
+    ...    Dasharo Tools Suite.
+    Write Into Terminal    6
+    Read From Trminal Until    Enter an option:
+    Write Into Terminal    1
+    ${output}=    Read From Terminal Until    shut down
+    Should Contain X Times    ${output}    VERIFIED    2
+    Sleep    10s
+
+Check Power Off In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    option for power off the DUT works correctly..
+    Sleep    5s
+    ${output}=    Read From Terminal
+    Length Should Be    ${output}    1
+
+Flash Firmware In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    ability for flashing firmware work correctly.
+    Execute Command In Terminal
+    ...    wget -O /tmp/coreboot.rom ${FW_DOWNLOAD_LINK}
+    ${out}=    Execute Command In Terminal
+    ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
+    ...    timeout=320s
+    ${verified}=    Run Keyword And Return Status
+    ...    Should Contain    ${out}    VERIFIED
+    IF    ${verified} == ${FALSE}
+        ${out}=    Execute Command In Terminal
+        ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
+        ...    timeout=320s
+        Should Contain    ${out}    identical
+    END
+
+Flash EC Firmware In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    ability for flashing EC firmware work correctly.
+    Execute Command In Terminal
+    ...    command=wget -0 /tmp/ec.rom https://3mdeb.com/open-source-firmware/Dasharo/${EC_BINARY_LOCATION}
+    ...    timeout=320s
+    ${output}=    Execute Command In Terminal
+    ...    command=system76_ectool flash ec.rom
+    ...    timeout=320s
+    Should Contain    ${output}    Successfully programmed SPI ROM
+    Sleep    10s
+
+Check Firmware Version
+    [Documentation]    Keyword allows to check firmware version in the Dasharo
+    ...    Tools Suite Shell.
+    ${output}=    Execute Command In Terminal    dmidecode -t 0
+    Should Contain    ${output}    ${FW_VERSION}
+
+Check EC Firmware Version
+    [Documentation]    Keyword allows to check EC firmware version in the
+    ...    Dasharo Tools Suite Shell.
+    ${output}=    Execute Command In Terminal    system76_ectool info
+    Should Contain    ${output}    ${EC_VERSION}
+
+Fwupd Update
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    ability for update firmware with the use of fwupd works correctly.
+    ${output}=    Execute Command In Terminal
+    ...    command=fwupdmgr refresh
+    ...    timeout=320s
+    Should Contain    ${output}    Successfully
+    ${output}=    Execute Command In Terminal
+    ...    command=fwupdmgr update
+    ...    timeout=320s
+    Should Contain    ${output}    Successfully installed firmware
+
+Get Firmware Version From Tianocore Setup Menu
```

Comment:
Also should not be in dts-lib.robot. Perhaps in menus.

Reply:
Moved to [menus](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/44ad73e6c5e5fe00e30995617fa9463cd758d089#diff-6ad7c5fb97cb1099c7ecd42f15dde3caf06d38b9fe5244afb96394454b6d6bfaR528)